pkg/controller/kabaneroplatform/landing.go

```diff
@@ -74,6 +76,11 @@ func deployLandingPage(_ context.Context, k *kabanerov1alpha2.Kabanero, c client
 		return err
 	}
 
+	err = addTLSConfigToLandingRoute(k, c, rev)
```

Comment:
I think we'll need to see if we're using version `0.1` of the orchestration here, and if we are, skip this step.